session7/src/com/itschool/homework/HomeRunChallenge7.java

```diff
@@ -0,0 +1,29 @@
+package com.itschool.homework;
+
+import java.util.Scanner;
+
+public class HomeRunChallenge7 {
+    public static void main(String[] args) {
+        HomeRunChallenge7 home = new HomeRunChallenge7();
+        home.challengeRun7();
+    }
+
+    public void challengeRun7() {
+        Scanner input = new Scanner(System.in);
+        int a = 20;
```

Comment:
this can be a constant, since it has always the same value, and maybe rename to something more suggestive, for example `LUCKY_NUMBER` or something like that.